### PR TITLE
Late9 keymaps update, added VIA support

### PR DESCRIPTION
**Pull request by rookiebwoy**
## Description

- removed multitap keymap
- created via keymap
- changed pid and vid in info.json
- updated readme file to better fit the template
- better comments overall

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/rookiebwoy/late9/readme.md
Comment (drashna) [inline keyboards/rookiebwoy/late9/readme.md:1]: Most of the changes here should be reverted

File: keyboards/rookiebwoy/late9/rules.mk
Comment (drashna) [inline keyboards/rookiebwoy/late9/rules.mk:1]: Revert this

File: lib/chibios-contrib
Comment (drashna) [inline lib/chibios-contrib:1]: revert this

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/rookiebwoy/late9/readme.md
Comment (rookiebwoy) [reply keyboards/rookiebwoy/late9/readme.md:1]: Sorry, I've made a mess trying to rebase the branch from `qmk:master` to `qmk:develop`.

File: lib/chibios-contrib
Comment (rookiebwoy) [reply lib/chibios-contrib:1]: Reverted to the commit before my rebase.

**Review by drashna — APPROVED** [review]

File: keyboards/rookiebwoy/late9/rev1/keymaps/via/rules.mk [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# VIA requested options
+
+VIA_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/rookiebwoy/late9/rev1/keymaps/via/rules.mk:3]:
```suggestion
VIA_ENABLE = yes
```

File: keyboards/rookiebwoy/late9/rev1/keymaps/via/readme.md
Comment (fauxpark) [inline keyboards/rookiebwoy/late9/rev1/keymaps/via/readme.md:1]: This file can be deleted.

File: keyboards/rookiebwoy/late9/rev1/rules.mk
Comment (fauxpark) [inline keyboards/rookiebwoy/late9/rev1/rules.mk:1]: Might as well move all of these over to info.json.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/rookiebwoy/late9/rev1/rules.mk
Comment (rookiebwoy) [reply keyboards/rookiebwoy/late9/rev1/rules.mk:1]: Is this mandatory? I'm used to keep feature enable/disable option in this file and use info.json only for data-driven info.

Comment (fauxpark) [reply keyboards/rookiebwoy/late9/rev1/rules.mk:1]: These are data driven as well.
https://docs.qmk.fm/#/reference_info_json?id=firmware-configuration

Comment (rookiebwoy) [reply keyboards/rookiebwoy/late9/rev1/rules.mk:1]: I understand. I have the same structure of this project in my [other one](https://github.com/qmk/qmk_firmware/blob/develop/keyboards/rookiebwoy/neopad/rev1/rules.mk). Can I keep it for consistence?

I politely ask what kinds of feature should I write in rules.mk, only the ones listed [here](https://github.com/qmk/qmk_firmware/blob/master/docs/config_options.md#the-rules.mk-file)?

Comment (rookiebwoy) [reply keyboards/rookiebwoy/late9/rev1/rules.mk:1]: It's fine to me. The only feature that seems to stay in the rules.mk is OLED_ENABLE (according to [docs](https://docs.qmk.fm/#/feature_oled_driver) ). Am I right?

File: keyboards/rookiebwoy/late9/rev1/rules.mk [hunk 2]
```diff
@@ -1,14 +1,3 @@
...
-MOUSEKEY_ENABLE = no        # Mouse keys
-EXTRAKEY_ENABLE = no        # Audio control and System control
-CONSOLE_ENABLE = no         # Console for debug
-COMMAND_ENABLE = no         # Commands for debug and configuration
-NKRO_ENABLE = yes           # Enable N-Key Rollover
-BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
-RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
-AUDIO_ENABLE = no           # Audio output
 
 OLED_ENABLE = yes
```
Comment (zvecr) [inline keyboards/rookiebwoy/late9/rev1/rules.mk:3]:
```make
# This file intentionally left blank
```

File: keyboards/rookiebwoy/late9/rev1/info.json [hunk 3]
```diff
@@ -19,6 +19,10 @@
         "rows": ["B4", "E6", "D7", "B3", "B2", "B6"]
     },
     "diode_direction": "ROW2COL",
+    "features": {
+      "bootmagic": true,
+      "nkro": true
```
Comment (zvecr) [inline keyboards/rookiebwoy/late9/rev1/info.json:24]:
```suggestion
      "nkro": true,
      "oled": true
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/rookiebwoy/late9/rev1/info.json (hunk 3 above)
Comment (rookiebwoy) [reply keyboards/rookiebwoy/late9/rev1/info.json]: Done. I think that a list of possible features in the documentation could be useful.

**Review by zvecr — APPROVED** [review]

**Pull request merged**